hail/src/test/scala/is/hail/SparkSuite.scala

```diff
@@ -29,7 +30,8 @@ class SparkSuite extends TestNGSuite {
 
   def sc: SparkContext = hc.sc
 
-  def initializeHailContext() {
+  @BeforeClass def ensureHailContextInitialized() {
```

Comment:
Also, I know there were some places in IRSuite or TableIRSuite if that exists where I called `hc` just to make sure it was initialized. Would be nice to get rid of those if you can easily find them.

Reply:
I think they were all calling `initializeHailContext` and I deleted them.  I think I got them all.